packages/paste-icons/src/ProductTwimlBinsIcon.tsx

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
This file name has incorrect capitalization. Can you rename it to `ProductTwiMLBinsIcon.tsx` please?

Renaming files in git on osx is awkward, you might need this command: `git mv ProductTwimlBinsIconProps.tsx ProductTwiMLBinsIconProps.tsx` for git to recognize the new filename

Reply:
Super weird... it was the correct name in my text editor and when I would `ls` all of the files in that folder, but I ran the command you suggested above, and looks like it changed the name 👍

Reply:
Yeah it's an issue with macs. You have to do the `git mv` command if you're renaming a file that is already checked in by just changing capitalization. Thanks!